test/Microsoft.Framework.ConfigurationModel.Test/IniFileConfigurationSourceTest.cs

```diff
@@ -212,8 +212,7 @@ public void IniConfiguration_Throws_On_Missing_Configuration_File()
                 catch (FileNotFoundException exception)
                 {
                     Assert.Equal(
-                        string.Format(Resources.Error_FileNotFound,
-                        Path.Combine(Directory.GetCurrentDirectory(), "NotExistingConfig.ini")),
+                        string.Format(Resources.Error_FileNotFound, "NotExistingConfig.ini"),
```

Comment:
nit: up to you but could also use `Resources.FormatError_FileNotFound("NotExistingConfig.ini")` here and in the `XmlConfigurationSourceTest` change.